src/services/user.js

```diff
@@ -315,6 +315,46 @@ const getMemberships = async id => {
   memberships = await http.get(`memberships/student/${id}`);
   return memberships;
 };
+
+const getTranscriptInfo = async id => {
+  let transcriptInfo = await getMemberships(id);
+  console.log(transcriptInfo);
+  transcriptInfo.activities = [0];
+  let k = 0;
```

Comment:
There are a lot of loop variables here (4).  Could you explain what they are for?  

"k" is puzzling me the most.